docs/pdk.md

```diff
@@ -2,66 +2,71 @@
 title = "PDK"
 weight = 11
 nav = [
+    "Examples and Executables",
     "Library",
-    "Examples",
 ]
 +++
 
 ## PDK
 
-The [Pilosa Dev Kit](https://github.com/pilosa/pdk) contains Go libraries to help you use Pilosa effectively. From importing data quickly, to managing the mappings from contiguous integer ids to values of other types, the PDK should help you get off the ground quickly.
+The [Pilosa Dev Kit](https://github.com/pilosa/pdk) contains executables, examples, and Go libraries to help you use Pilosa effectively.
+
+### Examples and Executables
+Running `pdk -h` will give the most up to date list of all the tools and examples that PDK provides. We'll cover a few of the more important ones here.
+
+#### Kafka
+`pdk kafka` reads either JSON or Avro encoded records from Kafka (using the
+Confluent Schema Registry in the case of Avro), and indexes them in Pilosa. Each
+record from Kafka is assigned a Pilosa column, and each value in a record is
+assigned a row or field. Frame and field names are built from the "path" through
+the record to arrive at that field. For example:
+
+```json
+{
+  "name": "jill",
+  "favorite_foods": ["corn chips", "chipotle dip"],
+  "location": {
+    "city": "Austin",
+    "state": "Texas",
+    "latitude": 3754,
+    "longitude": 4526
+  },
+  "active": true,
+  "age": 27
+}
+```
+
+This JSON object would result in the following Pilosa schema:
+
+| Name           | Field     | Type   | Size/Min |        Max |
+|----------------|-----------|--------|----------|------------|
+| name           |           | ranked |   100000 |            |
+| favorite_foods |           | ranked |   100000 |            |
+| default        |           | Ranked |   100000 |            |
+|                | age       | int    |        0 | 2147483647 |
+| location       |           | ranked |     1000 |            |
+|                | latitude  | int    |        0 | 2147483647 |
+|                | longitude | int    |        0 | 2147483647 |
+| location-city  |           | ranked |   100000 |            |
+| location-state |           | ranked |   100000 |            |
```

Comment:
I understand what this table is saying, but it might be more clear to readers if the size/min column was split into two. Alternatively, one separate table for fields.

Reply:
I agree - will follow up in separate PR